src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +22,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        return user.getPassword().equals(password);
```

Comment:
let's simplify it this way:
```suggestion
        return user != null && user.getPassword().equals(password);
```